spec/services/idv/session_spec.rb

```diff
@@ -161,14 +166,17 @@
           subject.create_profile_from_applicant_with_password(user.password)
           profile = subject.profile
 
-          expect(subject).not_to have_received(:move_pii_to_user_session)
           expect(profile.activated_at).to eq nil
           expect(profile.active).to eq false
           expect(profile.in_person_verification_pending?).to eq(true)
           expect(profile.fraud_review_pending?).to eq(false)
           expect(profile.gpo_verification_pending_at.present?).to eq false
           expect(profile.initiating_service_provider).to eq nil
           expect(profile.verified_at).to eq nil
+
+          pii_from_sesison = Pii::Cacher.new(user, user_session).fetch
+          expect(pii_from_sesison).to_not be_nil
+          expect(pii_from_sesison.ssn).to eq(Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE[:ssn])
```

Comment:
same as above and in three other places
```suggestion
          pii_from_session = Pii::Cacher.new(user, user_session).fetch
          expect(pii_from_session).to_not be_nil
          expect(pii_from_session.ssn).to eq(Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE[:ssn])
```